tests/api/api/test_grafana_proxy.py

```diff
@@ -0,0 +1,170 @@
+import os
+from typing import Optional
+
+import pytest
+from fastapi.testclient import TestClient
+from pytest import fail
+from sqlalchemy.orm import Session
+from v3io.dataplane import RaiseForStatus
+from v3io_frames import CreateError
+from v3io_frames import frames_pb2 as fpb2
+
+from mlrun.api.api.endpoints.grafana_proxy import (
+    _parse_query_parameters,
+    _validate_query_parameters,
+)
+from mlrun.api.api.endpoints.model_endpoints import (
+    ENDPOINTS_TABLE_PATH,
+    ENDPOINT_EVENTS_TABLE_PATH,
+)
+from mlrun.config import config
+from mlrun.errors import MLRunBadRequestError
+from mlrun.utils.v3io_clients import get_v3io_client, get_frames_client
+from tests.api.api.test_model_endpoints import (
+    _mock_random_endpoint,
+    _write_endpoint_to_kv,
+)
+
+ENV_PARAMS = {"V3IO_ACCESS_KEY", "V3IO_API", "V3IO_FRAMESD"}
+
+
+def _build_skip_message():
+    return f"One of the required environment params is not initialized ({', '.join(ENV_PARAMS)})"
+
+
+def _is_env_params_dont_exist() -> bool:
+    return not all((os.environ.get(r, False) for r in ENV_PARAMS))
+
+
+@pytest.mark.skipif(
+    _is_env_params_dont_exist(), reason=_build_skip_message(),
+)
+def test_grafana_proxy_model_endpoints_check_connection(
+    db: Session, client: TestClient
+):
+    response = client.get(
+        url="/api/grafana-proxy/model-endpoints",
+        headers={"X-V3io-Session-Key": _get_access_key()},
+    )
+    assert response.status_code == 200
```

Comment:
This endpoint can be definitely unit tested (you don't really do anything with the access key in the endpoint logic, so it can be as well `some-fake-access-key`)
You can (and probably should) leave also this "intergation test" version (which uses a real access key)